search_ieee.py

```diff
@@ -0,0 +1,56 @@
+import requests
+import logging
+from objects import Ieee
+
+logger = logging.getLogger('nfdi_search_engine')
+
+def ieee(search_term, results):
+    api_url = "https://ieeexploreapi.ieee.org/api/v1/search/articles"
+    api_key = "API_KEY"
+
+    params = {
+        "apikey": api_key,
+        "querytext": search_term
+    }
+
+    try:
+        # response = requests.get(api_url, params=params)
+        # response.raise_for_status()
+        # data = response.json()
+
+        #This data is only for testing purposes as a sample. It will be replaced once the IEEE API key is activated 
+        data = {
+            "articles": [
+                {
+                    "title": "[Sample article] An Overview of Search Engine Optimization",
+                    "authors": [
+                        {
+                            "full_name": "Varsha; P.S. Grover; Laxmi Ahuja"
+                        }
+                    ],
+                    "doi": "https://ieeexplore.ieee.org/document/9596287",
+                    "datepublished": "2021"
+                    
+                }
+            ]
+        }
+        for article in data["articles"]:
```

Comment:
This is a static value. The IEEE object is not updated. It always display the same result.

Reply:
As mentioned in the pull request description, this is a static record only intended as a temporary sample. Once the API key is activated, actual data will replace the static record.